extensions/oidc/runtime/src/main/java/io/quarkus/oidc/Bearer.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.oidc;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import io.quarkus.vertx.http.runtime.security.HttpAuthMechanism;
+
+/**
+ * Selects {@link io.quarkus.oidc.runtime.BearerAuthenticationMechanism}.
+ * Equivalent to '@HttpAuthMechanism("Bearer")'.
+ *
+ * @see HttpAuthMechanism for more information
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE, ElementType.METHOD })
+public @interface Bearer {
```

Comment:
```suggestion
public @interface BearerTokenAuthentication {
```

Proposing to be more informative, also, can it make sense to share these annotations, say in quarkus-security ? Because smallrye-jwt can also do the bearer token one

Reply:
>  can it make sense to share these annotations, say in quarkus-security ? 

IMO we should keep loose coupling.  The Quarkus Security is not aware of these mechanisms. I wonder why I didn't do anything about the Quarkus SmallRye JWT, probably because the PR is already big enough.

> Because smallrye-jwt can also do the bearer token one

I can't solve it without introducing a new SmallRye / OIDC SPI module (which I take it is not desirable). Is it problem to have `BearerTokenAuthentication` in the Quarkus SmallRye JWT package as well? It's different package.

Reply:
1. I've addressed renaming
2. HTTP Authentication mechanism is Vert.x HTTP based and the `quarkus-security` is not aware of it.

Reply:
@michalvavrik I think another package for smallrye-jwt will do, so please duplicate `BearerTokenAuthentication` there and it will be ready to go

Reply:
The `quarkus-integration-test-management-interface-auth` is already testing this for configuration based selection so I added test for this there. Perfect place.

Reply:
Done.